<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>***********************************************************************
*
* Copyright (c) 2012-2025 Barbara Geller
* Copyright (c) 2012-2025 Ansel Sermersheim
*
* Copyright (C) 2015 The Qt Company Ltd.
* Copyright (c) 2012-2016 Digia Plc and/or its subsidiary(-ies).
* Copyright (c) 2008-2012 Nokia Corporation and/or its subsidiary(-ies).
*
* This file is part of CopperSpice.
*
* CopperSpice is free software. You can redistribute it and/or
* modify it under the terms of the GNU Lesser General Public License
* version 2.1 as published by the Free Software Foundation.
*
* CopperSpice is distributed in the hope that it will be useful,
* but WITHOUT ANY WARRANTY; without even the implied warranty of
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
*
* https://www.gnu.org/licenses/
*
***********************************************************************</comment>
 <class>TranslateDialog</class>
 <widget class="QDialog" name="TranslateDialog">
  <property name="whatsThis">
   <string>This window allows you to search for some text in the translation source file.</string>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>174</height>
   </rect>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ledTranslateTo">
         <property name="whatsThis">
          <string>Type in the text to search for.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="findWhat">
         <property name="text">
          <string>Find &amp;source text:</string>
         </property>
         <property name="buddy">
          <cstring>ledFindWhat</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="translateTo">
         <property name="text">
          <string>&amp;Translate to:</string>
         </property>
         <property name="buddy">
          <cstring>ledTranslateTo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ledFindWhat">
         <property name="whatsThis">
          <string>Type in the text to search for.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Search options</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="ckMatchCase">
          <property name="text">
           <string>Match &amp;case</string>
          </property>
          <property name="whatsThis">
           <string>Texts such as 'TeX' and 'tex' are considered as different when checked.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ckMarkFinished">
          <property name="text">
           <string>Mark new translation as &amp;finished</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="findNxt">
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Find Next</string>
       </property>
       <property name="whatsThis">
        <string>Click here to find the next occurrence of the text you typed in.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="translate">
       <property name="text">
        <string>Translate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="translateAll">
       <property name="text">
        <string>Translate All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="whatsThis">
        <string>Click here to close this window.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>51</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ledFindWhat</tabstop>
  <tabstop>ledTranslateTo</tabstop>
  <tabstop>findNxt</tabstop>
  <tabstop>translate</tabstop>
  <tabstop>translateAll</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>ckMatchCase</tabstop>
  <tabstop>ckMarkFinished</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
